docs/Hooks.rst

```diff
@@ -144,8 +255,8 @@ Example hook to replace ``--with-verbs`` with ``--without-verbs`` in OpenMPI con
 
 .. code:: python
 
-    def pre_configure_hook(self, *args, **kwargs):
-        "Example pre-configure hook to replace --with-verbs with --without -verbs."""
-        if self.name == 'OpenMPI' and '--with-verbs' in self.cfg['configopts']:
-            self.log.info("[pre-configure hook] Replacing --with-verbs with --without-verbs")
-            self.cfg['configopts'] = self.cfg['configopts'].replace('--with-verbs', '--without-verbs')
+    def parse_hook(ec, *args, **kwargs):
+        "Example parse hook to replace --with-verbs with --without -verbs."""
+        if ec.name == 'OpenMPI' and '--with-verbs' in ec['configopts']:
+            ec.log.info("[pre-configure hook] Replacing --with-verbs with --without-verbs")
+            ec['configopts'] = ec['configopts'].replace('--with-verbs', '--without-verbs')
```

Comment:
Not sure that this example should be moved to parse_hook since it works just as well in pre_configure_hook.
Perhaps add a parse_hook example that changes a dependency or adds a patch or something.